make/wag.mk

```diff
@@ -1,23 +1,34 @@
 # This is the default Clever Wag Makefile.
 # Please do not alter this file directly.
-WAG_MK_VERSION := 0.1.0
+WAG_MK_VERSION := 0.2.0
 SYSTEM := $(shell uname -a | cut -d" " -f1 | tr '[:upper:]' '[:lower:]')
+WAG_INSTALLED := $(shell [[ -e "bin/wag" ]] && bin/wag --version)
+WAG_LATEST := $(shell curl -S https://api.github.com/repos/Clever/wag/releases/latest 2>/dev/null | grep tag_name | cut -d\" -f4)
```

Comment:
I don't think -S does anything here since you don't have -s. The best thing to do is probably to switch out -S for -s and then you shouldn't need the 2>/dev/null either.